projects/fal/pyproject.toml

```diff
@@ -24,6 +24,7 @@ grpc-interceptor = "^0.15.0"
 colorama = "^0.4.6"
 portalocker = "^2.7.0"
 rich = "^13.3.2"
+rich_click = "*"
```

Comment:
This didn't affect poetry.lock? Did we already have it there?

Reply:
@mederka Sorry, that's on me. Will update.

Reply:
@mederka Thanks again for the heads up. Added some hooks https://github.com/fal-ai/fal/pull/137